### DISCUSSION TITLE
Lesson 5: Brownie Test Issue

### DISCUSSION THREAD

**Discussion by ID-Mahone** [smartcontractkit/full-blockchain-solidity-course-py#420] [Q&A] [answered]
Can anyone help me with this problem please?
<img width="1416" alt="Bildschirmfoto 2021-11-22 um 02 41 42" src="https://user-images.githubusercontent.com/87675992/142789395-f60c740e-9b08-4b23-b48f-72a62c49a778.png">

I don't know how to fix this 'Import Brownie' issue.......
<img width="768" alt="Bildschirmfoto 2021-11-22 um 02 47 40" src="https://user-images.githubusercontent.com/87675992/142789569-a99d8eb0-30d5-4f9f-9d6b-47d1fe375e73.png">

**Comment by cromewar** [comment, marked as answer]
Hello @Maninng it says that you have an indentation error which means some part of your code is not correctly formatted, use ` Shift + Option + F` on Mac to Format the code properly, also pay attention to the import statement indentation as the error describes, if you can see on your screenshot you have a space " " just before the word `import`.

Regarding to the brownie error, this is because python vscode extension is incapable to recognize packages installed with pipx, you can delete eth-brownie and pipx, and install it again using just pip. 

Give it a try, best regards Cromewar.

**Comment by ID-Mahone** [comment]
With pip it is not working at all... But isn't Patrick also using pipx in vscode? I'm really stucked here :D

**Reply by cromewar** [reply]
Yes he is, but Patrick have the linting deactivated most of the course, anyways the brownie "Problem" is more a warning, even though vscode complain about, it will compile anyway but you will have that warning always.

You have to options:
1. Ignore the linting warning and use pipx
2. install with pip as I suggested but you'll probably need cython libraries you must install manually, with xcode and some stuff. The decision is yours.

Note: if you don't care having vscode complaining about brownie you can use pipx, the real problem in your code was the indentation actually.

**Reply by ID-Mahone** [reply]
Thank you a lot my friend i think i figured it out now.

**Reply by cromewar** [reply]
I'm glad it helped
